pkg/addons/addons.go

```diff
@@ -70,7 +69,10 @@ func GetAddonConfig(addons embed.FS, provider, addon string) (AddonConfig, error
 		return AddonConfig{}, err
 	}
 
-	configBytes, err := fs.ReadFile(addons, path.Join(selectedAddonPath, "draft.yaml"))
+	addOnConfigPath := path.Join(selectedAddonPath, "draft.yaml")
+	log.Debugf("addOnConfig is: %s", addOnConfigPath)
```

Comment:
💡 These debug logs are for us in case users machine is corrupted? (No crews involved in this file?) 🥷🥷☕️

Reply:
yep the addonconfig is just the draft config that lists variable names from the templates, but doesn't have any values